shell/components/CruResource.vue

```diff
@@ -326,6 +338,9 @@ export default {
     },
 
     async showPreviewYaml() {
+      // Required to populate yaml comments and default values
+      await this.schema?.fetchResourceFields();
```

Comment:
given way functions are called (createResourceYaml) and the beforehook in showPreviewYaml it can  be done in both places. should be harmless on second call. no duplicate http requests noticed whilst dev